whitepaper/chapters/timesync.tex

```diff
@@ -35,7 +35,7 @@ \subsection{Gathering Samples}
 
 After the start up of a node is completed, the node (hereafter called \emph{local node}) selects partner nodes for performing a time synchronization round.
 
-For all selected partners, the local node sends out a request asking the partner for its current network time.
+For all selected partners, the local node sends out a request asking the partners for their current network time.
```

Comment:
consider:

For each selected partners, the local node sends out a request asking the partner for its current network time

Reply:
selected partner (no s) - sorry i had a typo before